core/settings.go

```diff
@@ -33,12 +33,13 @@ type Settings struct {
 	// Deprecated: Will be removed in v0.9!
 	EmailAuth EmailAuthConfig `form:"emailAuth" json:"emailAuth"`
 
-	GoogleAuth   AuthProviderConfig `form:"googleAuth" json:"googleAuth"`
-	FacebookAuth AuthProviderConfig `form:"facebookAuth" json:"facebookAuth"`
-	GithubAuth   AuthProviderConfig `form:"githubAuth" json:"githubAuth"`
-	GitlabAuth   AuthProviderConfig `form:"gitlabAuth" json:"gitlabAuth"`
-	DiscordAuth  AuthProviderConfig `form:"discordAuth" json:"discordAuth"`
-	TwitterAuth  AuthProviderConfig `form:"twitterAuth" json:"twitterAuth"`
+	GoogleAuth      AuthProviderConfig `form:"googleAuth" json:"googleAuth"`
+	FacebookAuth    AuthProviderConfig `form:"facebookAuth" json:"facebookAuth"`
+	GithubAuth      AuthProviderConfig `form:"githubAuth" json:"githubAuth"`
+	GitlabAuth      AuthProviderConfig `form:"gitlabAuth" json:"gitlabAuth"`
+	DiscordAuth     AuthProviderConfig `form:"discordAuth" json:"discordAuth"`
+	TwitterAuth     AuthProviderConfig `form:"twitterAuth" json:"twitterAuth"`
+	MicrosoftAdAuth AuthProviderConfig `from:"microsoftAdAuth" json:"microsoftAdAuth"`
```

Comment:
is `from:` a typo here? all other providers user `form:`

Reply:
oops. Should be fixed now.